src/dodal/devices/pressure_jump_cell_adc.py

```diff
@@ -0,0 +1,63 @@
+from enum import Enum
+from functools import partial
+
+from ophyd_async.core import (
+    ConfigSignal,
+    DeviceVector,
+    SignalR,
+    StandardReadable,
+)
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+from ophyd import ADComponent as ADC
+from ophyd import (
+    AreaDetector,
+    CamBase,
+    Component,
+    Device,
+    EpicsSignal,
+    HDF5Plugin,
+    OverlayPlugin,
+    ProcessPlugin,
+    ROIPlugin,
+    StatsPlugin,
+    Signal,
+    StatusBase,
+)
```

Comment:
Should use ophyd-async instead of ophyd implementation of things.

This then is an implementation of StandardDetector

Reply:
Updated to use ophyd-async in 5f9f808cbd5260e22fc69b09b8524aabc9ea0434.